pages/posts/newsletters.md

```diff
@@ -6,9 +6,13 @@ permalink: /newsletters/
 ---
 
 {% include scrolltop.html %}
-{% for post in site.posts %}{% if post.category == "newsletter" %}{% assign currentDate = post.date | date: "%Y" %}{% if currentDate != myDate %}{% unless forloop.first %}</ul>{% endunless %}
-<h1 style="margin-top:20px; margin-bottom:10px">{{ currentDate }}</h1>
-  <ul>{% assign myDate = currentDate %}{% endif %}
+
+<ul>
```

Comment:
Let's restore how this file was previously, merge changes from https://github.com/USRSE/usrse.github.io/pull/157, and then rebuild the additional newsletter here with the change to ensure that it renders okay!